xml/net_sync.xml

```diff
@@ -80,782 +74,369 @@
+       user/group ownership, file permissions, and symbolic links. </para>
      </listitem>
     </varlistentry>
     <varlistentry>
-     <term>?</term>
+     <term><option>-z</option></term>
      <listitem>
-      <para>
-       This file does not exist in CVS.
-      </para>
+      <para>Compress the transmitted data</para>
      </listitem>
     </varlistentry>
    </variablelist>
-   <para>
-    The status <literal>M</literal> indicates a locally modified file. Either
-    commit the local copy to the server or remove the local file and run the
-    update again. In this case, the missing file is retrieved from the server.
-    If you commit a locally modified file and the file was changed in the same
-    line and committed, you might get a conflict, indicated with
-    <literal>C</literal>.
+   <note>
+    <title>Trailing Slashes Count</title>
+    <para>
+     Rsync has a peculiar syntax when it comes to directory names: the
+     <emphasis>trailing slash</emphasis>. If you put a trailing slash after the
+    directory it means the <emphasis>content</emphasis> of that directory. If
+    you omit the slash, you mean the directory itself.
+   </para>
+   </note>
+  </sect2>
+  <sect2 xml:id="sec.net.sync.rsync.copy.local.files-and-dirs">
+   <title>Copying Files and Directories Locally</title>
+   <para> Assuming a directory <filename>/var/backup</filename>
+    with write permissions.
+    To copy a single file on your local machine from
+    one path to another path, use the following command: </para>
+   <screen>&prompt.user;<command>rsync</command> -avz backup.tar.xz /var/backup/</screen>
+   <para>The file <filename>backup.tar.xz</filename> is copied to
+     <filename>/var/backup/</filename>, the absolute path will be
+     <filename>/var/backup/backup.tar.xz</filename>.</para>
+   <para>
+    Keep in mind to insert the <emphasis>trailing slash</emphasis> after the
+    <filename>/var/backup/</filename> directory! If you do not insert the slash,
+    the file <filename>backup.tar.xz</filename> is copied to <filename>/var/backup</filename>
+    (file) <emphasis>not</emphasis> inside the directory <filename>/var/backup/</filename>!
+   </para>
+   <para>
+    To copy a directory is similar to copy a single file. The following
+    example copies the directory <filename>&exampleuser_plain;/</filename> and
+    its content into the directory <filename>/var/backup/</filename>: </para>
+    <screen>&prompt.user;<command>rsync</command> -avz &exampleuser_plain; /var/backup/</screen>
+    <para> Find the copy in the absolute path
+     <filename>/var/backup/&exampleuser_plain;/</filename>. </para>
+  </sect2>
+  <sect2 xml:id="sec.net.sync.rsync.copy.remote.files-and-dirs">
+   <title>Copying Files and Directories Remotely</title>
+   <para> To copy files from or to remote directories requires an IP address or
+    a domain name. A user name can be optional in case your current user name is
```

Comment:
your current user name is equal to the remote user name

**your current user names on the local and remote machine are the same**